frontend/src/views/ShootList.vue

```diff
@@ -314,9 +314,12 @@ export default {
       }
     },
     toggleFilter (key) {
-      if (this.showOnlyShootsWithIssues) {
-        const filters = this.getShootListFilters
-        this.setShootListFilter({ filter: key, value: !filters[key] })
+      const filters = this.getShootListFilters
+      this.setShootListFilter({ filter: key, value: !filters[key] })
+      this.$localStorage.setObject('shootListFilter', pick(filters, ['onlyShootsWithIssues', 'progressing', 'userIssues', 'deactivatedReconciliation', 'hideTicketsWithLabel']))
```

Comment:
I don't understand why this works? Is the correct value stored?

Reply:
maybe it's now clearer?